src/nrniv/partrans.cpp

```diff
@@ -247,7 +247,7 @@ static neuron::container::data_handle<double> non_vsrc_update(Node* nd,
             return p->param_handle(ix);
         }
     }
-    hoc_execerr_ext("partrans update: could not find parameter index (%d, %d) of %s",
+    hoc_execerr_fmt("partrans update: could not find parameter index (%d, %d) of %s",
```

Comment:
Does it accept C-style placeholders? Or only Python-style?

Reply:
does not :(